test/cypress/integration/card/card-details-page-validation.spec.js

```diff
@@ -0,0 +1,49 @@
+const cardPaymentStubs = require('../../utils/card-payment-stubs')
+
+const tokenId = 'be88a908-3b99-4254-9807-c855d53f6b2b'
+const chargeId = 'ub8de8r5mh4pb49rgm1ismaqfv'
+const createPaymentChargeStubs = cardPaymentStubs.buildCreatePaymentChargeStubs(tokenId, chargeId, 'en')
+const checkCardDetailsStubs = cardPaymentStubs.checkCardDetailsStubs(chargeId)
+
+describe('Card details page validation', () => {
+  beforeEach(() => {
+    // this test is for the full process, the session should be maintained
+    // as it would for an actual payment flow
+    Cypress.Cookies.preserveOnce('frontend_state')
+  })
+
+  it('Should setup the payment and load the page', () => {
+    cy.task('setupStubs', createPaymentChargeStubs)
+    cy.visit(`/secure/${tokenId}`)
+
+    cy.location('pathname').should('eq', `/card_details/${chargeId}`)
+    cy.window().its('chargeId').should('eq', `${chargeId}`)
+  })
+
+  it('Should enter card details', () => {
```

Comment:
Same as above

Reply:
we need this setup for this test to get to the card details page so we can test the errors